modules/win-autoscale/watchmaker-win-autoscale.cfn.json

```diff
@@ -629,6 +629,39 @@
                         }
                     },
                     "setup": {
+                        "commands": {
+                            "10-initialize-disks": {
+                                "command": {
+                                    "Fn::If": [
+                                        "CreateAppVolume",
+                                        {
+                                            "Fn::Join": [
+                                                "",
+                                                [
+                                                    {
+                                                        "Fn::FindInMap": [
+                                                            "ShellCommandMap",
+                                                            "powershell",
+                                                            "command"
+                                                        ]
+                                                    },
+                                                    " \"& { ",
+                                                    "$EC2LaunchInitDiskScript = \"${Env:ProgramData}\"+'\\Amazon\\EC2-Windows\\Launch\\Scripts\\InitializeDisks.ps1';",
+                                                    "$EC2LaunchInitDiskConfig = \"${Env:ProgramData}\"+'\\Amazon\\EC2-Windows\\Launch\\Config\\DriveLetterMappingConfig.json';",
+                                                    "if ([System.IO.File]::Exists(\"$EC2LaunchInitDiskScript\")) {",
```

Comment:
Any particular reason not to use `if (Test-Path $EC2LaunchInitDiskScript -and Test-Path $EC2LaunchInitDiskConfig)`?

Reply:
No particular reason.  I thought it was more aligned with other parts of the template.  I can switch to `Test-Path`.  I don't have a preference.

Reply:
Ahh, ok. Yeah, let's switch it... We generally only reach into the .NET methods directly when there is no equivalent in a PowerShell commandlet.